cerebrium/deployments/long-running-tasks.mdx

```diff
@@ -38,7 +37,7 @@ Later response:
   "result": {
     "text": "The concept of drilling to the center of Earth can be quite exciting but it faces a number of significant scientific and logistical challenges, most of which make the project currently impossible with current technology. But for an idea of what such a venture would entail, here is an imaginary plan:\n1. Planning and Research:\nYou'd need extensive research to understand the Earth's structure and composition. You'll need data on the Earth's core, how temperature and pressure increase with depth, current drilling technology, potential hazards and contingities.\n2. Site Selection:\nChoose a drilling site. Ideally, it would be a place with a relatively thin crust making the journey shorter. The oceanic crust is much thinner compared to the continental crust.\n3. Drill Design:\nYou might have to design a new drill to withstand extreme temperatures and pressures. The further down the Earth you go, the hotter and more pressurized it becomes. You would need a drill made from extremely heat and pressure-resistant materials, perhaps with in-built cooling mechanisms.\n4. Journey Begins:\nStart drilling. However, today's technology can only drill approximately 12 kilometers into the Earth's crust. It's a bit like the thickness of an eggshell compared to the radius of the egg. "
   },
-  "status_code": 202,
+  "status_code": 200,
```

Comment:
this would be 202 since there is a webhook_endpoint

Reply:
No thats on the first response not the second